src/main/java/com/smalaca/messagesender/service/MessageCrud.java

```diff
@@ -22,10 +24,10 @@ public Response createNew(MessageDto messageDto) {
         Message message = new MessageFactory().createFrom(messageDto);
 
         if (!messageRepository.exists(message)) {
-            message.setId("1");
+            message.setId(UUID.randomUUID().toString());
```

Comment:
tutaj powinieneś mieć zależność + interfejs + implementację. Stworzyłem na to osobne zadanie 😄